src/framework/grid/Grid.spec.js

```diff
@@ -103,21 +67,7 @@ describe('Grid', () => {
         };
 
         const testCase = TestCaseFactory.createFromElement(<Grid {...props} />);
```

Comment:
Unfortunately, `create` can only substitute for `createFromFunction` or `createFromClass`, but not `createFromElement`.